shell/src/test/java/org/apache/accumulo/shell/ShellSetInstanceTest.java

```diff
@@ -119,17 +120,15 @@ public void tearDown() {
     SiteConfiguration.clearInstance();
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
Dislike the warnings suppression. If it's testing a feature of mock, it should be marked deprecated itself (because it'll go away when what it's testing goes away).